src/net/sf/memoranda/ui/WorkPanel.java

```diff
@@ -259,9 +273,21 @@ public void eventsB_actionPerformed(ActionEvent e) {
 	}
 
 	public void filesB_actionPerformed(ActionEvent e) {
-		cardLayout1.show(panel, "FILES");
-		setCurrentButton(filesB);
-		Context.put("CURRENT_PANEL", "FILES");
+		if (Configuration.get("RESOURCE_PATH").equals("nothing")) {
+			JFileChooser picker = new JFileChooser();
+			picker.setFileSelectionMode(JFileChooser.DIRECTORIES_ONLY);
+			picker.showOpenDialog(picker);
+			File chosen = picker.getSelectedFile();
+			Configuration.put("RESOURCE_PATH", chosen.toString());
+		}
+		if(isWindows()){
+			try {
+				Desktop.getDesktop().open(new File((String) Configuration.get("RESOURCE_PATH")));
```

Comment:
We should be checking whether or not the host OS even supports `Desktop` operations (see `isDesktopSupported()`). If not, we should be defaulting to another cross-platform, user-friendly way of opening the resource file (maybe keep existing resource type architecture in place as backup?)